publication-doi-commons/src/main/java/no/unit/nva/publication/doi/update/dto/DoiUpdateHolder.java

```diff
@@ -5,7 +5,7 @@
 import nva.commons.json.JsonSerializable;
 
 public class DoiUpdateHolder implements JsonSerializable {
```

Comment:
maybe we should make it explicit somehow that this is an EventBridge event detail structure?

Reply:
`DoiUpdateStatusEventBridgeHolder` ?

Reply:
DoiUpdateStatusEventDetail maybe?

Reply:
I suggest we make a own task for this ( @brinxmat ) do this in a new PR, there is a lot of PRs up and we have to update a lot of feature branches to align the new holder data class.